### ISSUE TITLE
Getting a FileLoadException when manually running xunit against a test assembly

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn#6052]
__Symptom:__
When running xunit manually against a test binary, tests fail with something similar to:
```
      System.IO.FileLoadException : Could not load file or assembly 'Microsoft.CodeAnalysis.CSharp, Version=42.42.42.42, Culture=neutral, PublicKey
Token=31bf3856ad364e35' or one of its dependencies. Strong name signature could not be verified.  The assembly may have been tampered with, or it w
as delay signed but not fully signed with the correct private key. (Exception from HRESULT: 0x80131045)
```

__Cause:__
By default Roslyn builds with [Open Source Signing](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/oss-signing.md). This is incompatible with the CLR's Shadow Caching that xunit turns on by default.

__Resolution:__
Simply pass the -noshadow switch to the command line:
```
%USERPROFILE\.nuget\packages\xunit.runner.console\2.1.0\tools\xunit.console.x86.exe "Binaries\Debug\Roslyn.ExpressionEvaluator.CSharp.ResultProvider.UnitTests.dll" -noshadow
```

**Comment by tannergooding** [comment]
Closing as issue already has a resolution.

As additional context, this will be resolved (no need to pass -noshadow) when #6088 is merged.

**Comment by ashish2rathi** [comment]
@davkean : Can you please explain how to fire the given command, and from where?

%USERPROFILE\.nuget\packages\xunit.runner.console\2.1.0\tools\xunit.console.x86.exe "Binaries\Debug\Roslyn.ExpressionEvaluator.CSharp.ResultProvider.UnitTests.dll" -noshadow
